WMF Framework/Remote Notifications/RemoteNotificationsOperationsController.swift

```diff
@@ -122,3 +81,91 @@ class RemoteNotificationsOperationsController {
         }
     }
 }
+
+final class RemoteNotificationsOperationsTimeController {
+    weak var viewContext: NSManagedObjectContext?
```

Comment:
Is it better to keep this value on the `viewContext` or add `WMFKeyValue` to `RemoteNotifications.xcdatamodeld` (like in `EventLogging.xcdatamodeld`) and use the private context created by `RemoteNotificationsModelController`?

Reply:
It'd be better to use the private context